docs/src/man/basics.md

```diff
@@ -1565,40 +1565,1207 @@ julia> german[Not(5), r"S"]
-  stored in `target_column_name` column.
-- `source_column => transformation`; In this scenario we apply the
-  transformation function to `source_column` and the target column names is
-  automatically generated.
-- `source_column => target_column_name` renames the `source_column` to
-  `target_column_name`.
-- `source_column` just keep the source column as is in the result without any
-  transformation;
-
-These rules are typically called transformation mini-language.
-
-Let us move to the examples of application of these rules
+## Basic Usage of Manipulation Functions
+
+In DataFrames.jl there are seven functions that can be used
+to manipulate data frame columns:
+
+| Function     | Memory Usage                     | Column Retention                             | Row Retention                                     |
+| ------------ | -------------------------------- | -------------------------------------------- | ------------------------------------------------- |
+| `transform`  | Creates a new data frame.        | Retains both source and manipulated columns. | Retains same number of rows as source data frame. |
+| `transform!` | Modifies an existing data frame. | Retains both source and manipulated columns. | Retains same number of rows as source data frame. |
+| `select`     | Creates a new data frame.        | Retains only manipulated columns.            | Retains same number of rows as source data frame. |
+| `select!`    | Modifies an existing data frame. | Retains only manipulated columns.            | Retains same number of rows as source data frame. |
+| `subset`     | Creates a new data frame.        | Retains only source columns.                 | Number of rows is determined by the manipulation. |
+| `subset!`    | Modifies an existing data frame. | Retains only source columns.                 | Number of rows is determined by the manipulation. |
+| `combine`    | Creates a new data frame.        | Retains only manipulated columns.            | Number of rows is determined by the manipulation. |
+
+### Constructing Operation Pairs
+All of the functions above use the same syntax which is commonly
+`manipulation_function(dataframe, operation)`.
+The `operation` argument is a `Pair` which defines the
+operation to be applied to the source `dataframe`,
+and it can take any of the following common forms explained below:
+
+`source_column_selector`
+: selects source column(s) without manipulating or renaming them
+
+`source_column_selector => operation_function`
+: passes source column(s) as arguments to a function
+and automatically names the resulting column(s)
+
+`source_column_selector => operation_function => new_column_names`
+: passes source column(s) as arguments to a function
+and names the resulting column(s) `new_column_names`
+
+`source_column_selector => new_column_names`
+: renames a source column,
+or splits a column containing collection elements into multiple new columns
+
+!!! Note
+      The `source_column_selector`
```

Comment:
you only cannot do column renaming, but you can do selection:
```
julia> using DataFrames

julia> df = DataFrame(id=1:4, b=[true, false, true, false])
4×2 DataFrame
 Row │ id     b
     │ Int64  Bool
─────┼──────────────
   1 │     1   true
   2 │     2  false
   3 │     3   true
   4 │     4  false

julia> subset(df, :b)
2×2 DataFrame
 Row │ id     b
     │ Int64  Bool
─────┼─────────────
   1 │     1  true
   2 │     3  true
```

Reply:
Fixed it.